network/hybridNetwork.go

```diff
@@ -57,14 +63,21 @@ func NewHybridP2PNetwork(log logging.Logger, cfg config.Local, datadir string, p
 		tracker: identityTracker,
 		scheme:  NewIdentityChallengeScheme(NetIdentityDedupNames(cfg.PublicAddress, p2pnet.PeerID().String()), NetIdentitySigner(p2pnet.PeerIDSigner())),
 	}
-	wsnet, err := NewWebsocketNetwork(log, cfg, phonebookAddresses, genesisID, networkID, nodeInfo, &identOpts)
+	wsMeshCreator := meshCreator
+	if wsMeshCreator == nil && cfg.NetAddress != "" {
```

Comment:
Here is that blank check again.  But now it's the same check, but used with `ws` instead of `p2p`. I don't understand it, I guess.